src/controllers/tripsController.js

```
@@ -75,6 +78,7 @@ class tripsController {
         }
       });
     } catch (error) {
+      console.log(error);
```

@RusimbiP it's better to remove ```console.log```  statements in the codebase.

---

@hakoemmy What makes it better? What is your suggestion?

---

@Patrick for me I think you should return the error you are catching instead of logging it

---

@AbdoulNuru You're right